### PR TITLE
test(textarea): added rows textarea test and autosize textarea test

### LINKED ISSUE #1464: Enhance Test Coverage for `@yamada-ui/textarea`
package: @yamada-ui/textarea

### Description
`@yamada-ui/textarea` has a test coverage of less than 80%. We will enhance it to above 80%.

### Target files
- [packages/components/textarea/src/textarea.tsx](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/components/textarea/src/textarea.tsx)
  - [L99](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/components/textarea/src/textarea.tsx#L99)
  - [L113](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/components/textarea/src/textarea.tsx#L113)
  - [L130](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/components/textarea/src/textarea.tsx#L130)

### Problem Statement/Justification
Testing is essential to ensure quality when operating a package.

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.

### PR DESCRIPTION
**Pull request by Kei-Ta**
Closes #1464

## Description

Added Textarea tests for enhance test coverage.

## Current behavior (updates)

NIL

## New behavior

NIL

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: .changeset/tiny-garlics-bow.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (108yen) [inline .changeset/tiny-garlics-bow.md:1]: No `changeset` is required for testing.

**Review by 108yen — CHANGES_REQUESTED** [review]

File: .changeset/tiny-garlics-bow.md
Comment (illionillion) [inline .changeset/tiny-garlics-bow.md:1]: Changesets are only required for changes related to the implementation of components, and changes to test code do not require changesets.

Comment (Kei-Ta) [reply .changeset/tiny-garlics-bow.md:1]: @108yen @illionillion 
Thank you for your reviews.
Should I remove changeset and commit?

**Comment by Kei-Ta** [comment]
@108yen @illionillion
Thank you for your reviews.
Should I remove changeset and commit?

File: packages/components/textarea/tests/textarea.test.tsx [hunk 2]
```diff
@@ -38,4 +38,32 @@ describe("<Textarea />", () => {
 
     expect(screen.getByRole("textbox")).toHaveAttribute("placeholder", "text")
   })
+  test("Rows Textarea renders correctly", () => {
+    render(<Textarea rows={1} />)
```
Comment (108yen) [inline packages/components/textarea/tests/textarea.test.tsx:42]: The test passes even if `row` is removed.
```ts
    render(<Textarea />)
```

File: packages/components/textarea/tests/textarea.test.tsx [hunk 3]
```diff
@@ -38,4 +38,32 @@ describe("<Textarea />", () => {
...
+      "fonts" in global.document ? global.document.fonts : undefined
+
+    Object.defineProperty(global.document, "fonts", {
+      value: {
+        addEventListener: vi.fn(),
+        removeEventListener: vi.fn(),
+      },
+      writable: true,
+    })
+    render(<Textarea autosize />)
```
Comment (108yen) [inline packages/components/textarea/tests/textarea.test.tsx:59]: The test passes even if `autosize` is removed.
```ts
    render(<Textarea />)
```

**Comment by 108yen** [comment]
> @108yen @illionillion Thank you for your reviews. Should I remove changeset and commit?

Yes, please.

File: packages/components/textarea/tests/textarea.test.tsx (hunk 3 above)
Comment (Kei-Ta) [reply packages/components/textarea/tests/textarea.test.tsx:59]: Sorry, there wasn't enough testing.
I fixed test condition and remove changeset.

**Comment by 108yen** [comment]
@Kei-Ta 
Check that the `rows` property, not the `minHeight` style, is at the intended value.
```ts
    expect(screen.getByRole("textbox")).toHaveProperty("rows", 1)
```

**Comment by Kei-Ta** [comment]
@108yen 
OK, I fixed it.

> @Kei-Ta Check that the `rows` property, not the `minHeight` style, is at the intended value. For example.
> 
> ```ts
>     expect(screen.getByRole("textbox")).toHaveProperty("rows", 1)
> ```

**Comment by 108yen** [comment]
@Kei-Ta 
Thank you!
Could you test `autosize` in the same way?

**Comment by Kei-Ta** [comment]
@108yen 
I fixed autosize textarea test.
If your intention is different, please point it out.

**Review by 108yen — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**